surf/apps/materials/migrations/0019_auto_20191021_1326.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.0.13 on 2019-10-21 13:26
+
+from django.db import migrations
+import django_enumfield.db.fields
+import surf.apps.materials.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('materials', '0018_auto_20190830_0934'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='collection',
+            name='is_shared',
```

Comment:
Yes, let's kill this and remove the button. Only owners can share collections right now and that doesn't make sense really